omero/developers/Model/KeyValuePairs.rst

```diff
@@ -89,7 +89,7 @@ any of the OME types which are annotatable. Such annotations
 attached to key UI elements like images and datasets will be
 presented by the clients, and can be edited with the
 appropriate permissions. See :help:`Managing Data
-<managing-data.html#keyvalue>` on :help:`OMERO User Help<>`
+<introduction/docs/index.html>` on :help:`OMERO User Help<>`
```

Comment:
```suggestion
<introduction/docs/data-management.html>` on :help:`OMERO User Guides<>`
```

Hopefully this will work. Not sure what does the ``:help:`` do, so not replacing it.

Reply:
Maybe the `help` extlink should become `guide` for clarity?

Reply:
I was trying to reduce the change but I ended up changing most location
I could turn it into ``:guide:``